src/main/kotlin/no/nav/syfo/util/ExtractFromFellesFormat.kt

```diff
@@ -28,7 +31,7 @@ fun extractOrganisationHerNumberFromSender(fellesformat: XMLEIFellesformat): XML
         it.typeId.v == "HER"
     }
 
-fun extractOrganisationRashNumberFromSender(fellesformat: XMLEIFellesformat): XMLIdent? =
+fun extractOrganisationReshNumberFromSender(fellesformat: XMLEIFellesformat): XMLIdent? =
```

Comment:
Resh eller RSH?

Reply:
Vanligste forkortelse ser ut til å være RESH --> https://www.nhn.no/grunndata/register-for-enheter-i-spesialisthelsetjenesten-resh/